assets/js/models/easel.coffee

```diff
@@ -0,0 +1,38 @@
+@tacit ?= {}
```

Comment:
clearly i git-fcked up with these files because they should just be renames but instead it thinks i added the code...sorry :(